apps/barista-design-system/src/app/app.module.ts

```diff
@@ -30,8 +30,8 @@ import { DtAutocompleteModule } from '@dynatrace/barista-components/autocomplete
 import { DtTagModule } from '@dynatrace/barista-components/tag';
 import { DtThemingModule } from '@dynatrace/barista-components/theming';
 import { environment } from '../environments/environment';
-import { BaPageGuard } from '../shared/services/page-guard';
-import { BaPageService } from '../shared/services/page.service';
+import { BaPageGuard } from 'libs/shared/data-access-strapi/src/lib/page-guard';
+import { BaPageService } from 'libs/shared/data-access-strapi/src/lib/page.service';
```

Comment:
Use the tsconfig path alias for importing from a library.

```suggestion
import { BaPageGuard, BaPageService } from '@dynatrace/shared/data-access-strapi';
```